### PR TITLE
Add kbd19x 7u winkeyless 

### PR DESCRIPTION
**Pull request by fatb0y**
## Description

Added kbd19x Winkeyless Layout 
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kbdfans/kbd19x/kbd19x.h [hunk 1]
```diff
@@ -29,74 +29,84 @@ inline void kbd19x_sclk_led_off(void)   { DDRB &= ~(1<<1); PORTB &= ~(1<<1); }
...
+  {k100, k101, k102, k103, k104, k105, k106, k107, k108, k109, k110, k111, k112},\
+  {k200, k201, k202, k203, k204, k205, k206, k207, k208, k209, k210, k211, k212},\
+  {k300, k301, k302, k303, k304, k305, k306, k307, k308, k309, k310, k311, XXXX},\
+  {k400, XXXX, k402, k403, k404, k405, k406, k407, k408, k409, k410, k411, k412},\
+  {k500, k501, k502, k503, k504, k505, k506, k507, k508, k509, k510, k511, XXXX},\
+  {k600, k601, k602, k603, k604, k605, k606, k607, k608, k609, k610, k611, k612},\
+  {XXXX, k701, k702, k703, k704, k705, k706, k707, k708, k709, k710, k711, XXXX},\
+}
+
+#define LAYOUT_WKL( \
```
Comment (drashna) [inline keyboards/kbdfans/kbd19x/kbd19x.h:95]:
```suggestion
#define LAYOUT_wkl( \
```

File: keyboards/kbdfans/kbd19x/info.json [hunk 2]
```diff
@@ -216,6 +216,107 @@
                 {"label":".", "x":17.5, "y":5.5},
                 {"label":"Split Enter (Bottom)", "x":18.5, "y":5.5}
             ]
+        },
+     "LAYOUT_wkl7u": {
```
Comment (drashna) [inline keyboards/kbdfans/kbd19x/info.json:220]: this needs to match the actual layout name. 
```suggestion
     "LAYOUT_wkl": {
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/kbdfans/kbd19x/info.json [hunk 3]
```diff
@@ -216,6 +216,107 @@
                 {"label":".", "x":17.5, "y":5.5},
                 {"label":"Split Enter (Bottom)", "x":18.5, "y":5.5}
             ]
+        },
+     "LAYOUT_wkl": {
+            "key_count": "97",
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd19x/info.json:221]:
```suggestion
```

File: keyboards/kbdfans/kbd19x/config.h [hunk 4]
```diff
@@ -49,7 +49,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define DIODE_DIRECTION COL2ROW
 #define BACKLIGHT_PIN B6
 #ifdef BACKLIGHT_PIN
-#define BACKLIGHT_LEVELS 3
+#define BACKLIGHT_LEVELS 6
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd19x/config.h:52]:
```suggestion
#define BACKLIGHT_LEVELS 3
```

File: keyboards/kbdfans/kbd19x/kbd19x.h [hunk 5]
```diff
@@ -29,74 +29,84 @@ inline void kbd19x_sclk_led_off(void)   { DDRB &= ~(1<<1); PORTB &= ~(1<<1); }
...
+*/
 
 #define LAYOUT_all( \
-  k00,  k01, k02, k03, k04,  k60, k61, k62, k63,  k05, k06, k07, k08,   k72,  k09, k0a, k0b, k0c, \
-  k10, k11, k12, k13, k14, k64, k65, k66, k67, k15, k16, k17, k18, k70, k71,  k19, k1a, k1b, k1c, \
-  k20,  k21, k22, k23, k24, k68, k69, k6a, k6b, k25, k26, k27, k28,     k73,  k29, k2a, k2b, k2c, \
-  k30,    k31, k32, k33, k34, k6c, k75, k76, k77, k35, k36, k37,        k38,  k39, k3a, k3b, k3c, \
-  k40,  k41, k42, k43, k44, k78, k79, k7a, k7b, k45, k46, k47, k48,    k74,   k49, k4a, k4b, k4c, \
-  k50,  k51, k52,                k59,            k55, k56, k57,   k58, k53, k54,   k5a, k5b, k5c  \
+  k000, k001, k002, k003, k004, k600, k601, k602, k603, k005, k006, k007, k008,       k702,  k009, k010, k011, k012, \
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd19x/kbd19x.h:58]: Please undo the changes to these layout macros, it makes it very difficult to see the actual changes to this file.

**Review by fauxpark — CHANGES_REQUESTED** [review]
